Level,Message,Timestamp
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:26:47
ERROR,"Traceback (most recent call last):
",2025-01-27 19:26:49
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 78, in <module>
    data = mb.receive_asset_data(""EURUSD"")
",2025-01-27 19:26:49
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 72, in receive_asset_data
    candles.append(self._trade_mapper.map_candle(candle))
",2025-01-27 19:26:49
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/mappers/TradeMapper.py"", line 48, in map_candle
    raise MappingFailedExceptionError(""Candle"")
",2025-01-27 19:26:49
ERROR,"KeyboardInterrupt
",2025-01-27 19:26:49
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:29:44
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:29:46
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:31:11
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:33:22
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:35:06
ERROR,"Traceback (most recent call last):
",2025-01-27 19:36:53
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 81, in <module>
    data = mb.receive_asset_data(""EURUSD"")
",2025-01-27 19:36:53
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 75, in receive_asset_data
    continue
",2025-01-27 19:36:53
ERROR,"KeyboardInterrupt
",2025-01-27 19:36:53
ERROR,"Traceback (most recent call last):
",2025-01-27 19:51:02
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 85, in <module>
    data = mb.receive_asset_data(""EURUSD"")
",2025-01-27 19:51:02
ERROR,"KeyboardInterrupt
",2025-01-27 19:51:02
CRITICAL,Mapping of Object Failed: Candle,2025-01-27 19:53:05
ERROR,"Traceback (most recent call last):
",2025-01-27 19:53:05
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/mappers/AssetMapper.py"", line 65, in map_candle_from_trading_view
",2025-01-27 19:53:05
ERROR,"return Candle(asset, broker, open, high, low, close, iso_time, timeFrame)",2025-01-27 19:53:05
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/models/asset/Candle.py"", line 26, in __init__
",2025-01-27 19:53:05
ERROR,"raise TypeError(f""IsoTime must be a string or datetime, got {type(iso_time).__name__}"")",2025-01-27 19:53:05
ERROR,TypeError,2025-01-27 19:53:05
ERROR,: ,2025-01-27 19:53:05
ERROR,"IsoTime must be a string or datetime, got NoneType",2025-01-27 19:53:05
ERROR,"
During handling of the above exception, another exception occurred:

",2025-01-27 19:53:05
ERROR,"Traceback (most recent call last):
",2025-01-27 19:53:05
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/mappers/AssetMapper.py"", line 67, in map_candle_from_trading_view
",2025-01-27 19:53:05
ERROR,"raise MappingFailedExceptionError(""Candle"")",2025-01-27 19:53:05
ERROR,app.mappers.exceptions.MappingFailedExceptionError,2025-01-27 19:53:05
ERROR,.,2025-01-27 19:53:05
ERROR,MappingFailedExceptionError,2025-01-27 19:53:05
ERROR,: ,2025-01-27 19:53:05
ERROR,Mapping of Object Failed: Candle,2025-01-27 19:53:05
ERROR,"Traceback (most recent call last):
",2025-01-27 19:54:35
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 86, in <module>
    candle = am.map_candle_from_trading_view(candle_from_dict)
",2025-01-27 19:54:35
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/mappers/AssetMapper.py"", line 62, in map_candle_from_trading_view
    low = candle.get(""low"")
",2025-01-27 19:54:35
ERROR,"KeyboardInterrupt
",2025-01-27 19:54:35
ERROR,"Traceback (most recent call last):
",2025-01-27 19:55:13
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 85, in <module>
    for candle_from_dict in candles_dict:
",2025-01-27 19:55:13
ERROR,"KeyboardInterrupt
",2025-01-27 19:55:13
ERROR,"Traceback (most recent call last):
",2025-01-27 20:00:20
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 91, in <module>
    data = mb.receive_asset_data(""EURUSD"")
",2025-01-27 20:00:20
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 71, in receive_asset_data
    candles:list[Candle] = []
",2025-01-27 20:00:20
ERROR,"KeyboardInterrupt
",2025-01-27 20:00:20
ERROR,"Traceback (most recent call last):
",2025-01-27 20:03:50
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 91, in <module>
    data = mb.receive_asset_data(""BTCUSDT"")
",2025-01-27 20:03:50
ERROR,"  File ""/Users/lauris/PycharmProjects/AutomatedTrading/app/db/mongodb/mongoDBData.py"", line 78, in receive_asset_data
    return candles
",2025-01-27 20:03:50
ERROR,"KeyboardInterrupt
",2025-01-27 20:03:50
